blueocean-dashboard/src/main/js/components/ActivityDetailsRow.jsx

```diff
@@ -89,7 +89,7 @@ class ActivityDetailsRow extends Component {
                 <TableCell linkTo={runDetailsUrl}><RunIdCell run={run} /></TableCell>
                 <TableCell linkTo={runDetailsUrl}><CommitId commitId={run.commitId} /></TableCell>
                 { isMultibranch && <TableCell linkTo={runDetailsUrl}>{branchName}</TableCell> }
-                <TableCell><RunMessageCell linkTo={runDetailsUrl} run={run} t={t} /></TableCell>
+                <TableCell><RunMessageCell linkTo={runDetailsUrl} run={run} t={t} changesUrl={changesUrl}/></TableCell>
```

Comment:
@kzantow guesses the lack of space would be a lint fail - which would trip this up lol (NO BUNDLE FOR YOU!)

Reply:
yes, confirmed that it's a lint error in the logs and failed `mvntest`